### DISCUSSION TITLE
OpenSeesPy Plotting - Animated Load Displacement

### DISCUSSION THREAD

**Discussion by AmyMcCall** [u-anurag/OpenSeesPy#100] [Q&A]
Anurag,

Thank you for the work you have done on the OpenSeesPy visualization guide. Do you know if there is a method for creating an animated plot for load vs displacement? It would be similar to the animate_deformedshape in that the plot changes with time but instead of plotting the nodes a frame at a time, it would be a continuous plot showing the progress of the load/disp over time (see attached plot). 

If you have any ideas, your help would be appreciated. Thanks,

Amy
![Load Displacement Plot_FF08-1-SHI000](https://user-images.githubusercontent.com/99770560/167492645-25d8085a-1989-424d-a28c-bb0a512c699f.jpg)

**Comment by u-anurag** [comment]
Thank you, @AmyMcCall for using the tool. I have planned to add something similar but it is not a priority for me. You should be able to implement matplotlib animation for a simple graph for your own work. There are tutorials available. 

I have moved the visualization library to a separate repository [VFO](https://github.com/u-anurag/vfo). All the new code will be added there. OpenSeesPy code will not carry any visualization support.

**Reply by AmyMcCall** [reply]
Thank you for your reply and suggestion. I have been able to find some good examples in the matplotlib. Thanks. If you ever develop a more elaborate animated load-disp plot, keep me posted.

**Reply by u-anurag** [reply]
Sure, you can start following the VFO repository so you get the updates.
